src/components/Pages/MyAssets/Pools/CreatePool.tsx

```diff
@@ -37,10 +37,13 @@ const CreatePool = () => {
     async (event: FormEvent) => {
       event.preventDefault();
       const selectedPoolToken = availablePoolTokens?.[Number(newPoolToken)];
-      if (api && selectedPoolToken) {
+      if (api && selectedPoolToken && nativeBalance) {
         // validate user has enough funds to cover the setup fee
         const poolSetupFee = api.consts.assetConversion?.poolSetupFee ?? null;
         const existentialDeposit = api.consts.balances.existentialDeposit;
+
+        // TODO check later if we can add a proper type for poolSetupFee
```

Comment:
What if we put `const poolSetupFee: BN | null`?

Reply:
tried that
Type 'Codec' is missing the following properties from type 'BN': clone, toNumber, toArray, toArrayLike, and 82 more.